### PR TITLE
mympd: 6.10.0 -> 7.0.2

### PR DESCRIPTION
**Pull request by Kranzes**
###### Motivation for this change
Bump to latest version at the time of making this RP.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120273` at 379a37ff run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>mympd</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `mongoose`.
Near pkgs/applications/audio/mympd/default.nix:11:3:
```
   |
11 | , mongoose
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/audio/mympd/default.nix [hunk 1]
```diff
@@ -9,17 +9,18 @@
 , libid3tag
 , flac
 , mongoose
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/mympd/default.nix:11]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by fufexan** [comment]
Result of `nixpkgs-review pr 120273` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mympd</li>
  </ul>
</details>

**Comment by doronbehar** [comment]
Thanks!

**Pull request merged**